src/components/Menu/MenuDivider/MenuDivider.module.scss

```
@@ -1,6 +1,8 @@
 @import "../../../styles/global-css-settings";
 
 .menuChildDivider {
-  margin-left: calc(var(--spacing-small) * -1);
-  width: calc(100% + var(--spacing-medium));
+  margin-left: 0px;
+  width: 100%;
+  padding-left: 8px;
+  padding-right: 8px;
```

Looks well, thanks! I suggest just a bit clearer version of the code
```suggestion
  width: 100%;
  padding: 0 var(--spacing-small);
```